client/testutil/driver_compatible.go

```diff
@@ -14,10 +14,7 @@ func ExecCompatible(t *testing.T) {
 }
 
 func QemuCompatible(t *testing.T) {
-	if runtime.GOOS == "windows" {
-		t.Skip("Must be on non-windows environments to run test")
-	}
-	// else see if qemu exists
+	// Check if qemu exists
 	_, err := exec.Command("qemu-system-x86_64", "-version").CombinedOutput()
```

Comment:
It still works because that binary returns without an exit code but it is confusing. Updated